sum.py

```diff
@@ -0,0 +1,6 @@
+sum=0;//variable to store sum
```

Comment:
Leave a comment block about the file created date and author something like this :
'''
Created on October 22, 2018
@author: anoopbhatn
'''

After this, specify the purpose of the code in a comment block